interface/globals.php

```diff
@@ -192,6 +192,15 @@ function undoMagicQuotes($array, $topLevel=true) {
 // Include the version file
 require_once (dirname(__FILE__) . "/../version.php");
 
+// The logging level for common/logging/logger.php
+$GLOBALS["log_level"] = "DEBUG"; // (value can be TRACE, DEBUG, INFO, WARN, or ERROR)
+
+// Should Doctrine run in "dev mode"?
+$GLOBALS["doctrine_dev_mode"] = false;
+
+// Should Doctrine make use of connection pooling?
+$GLOBALS["doctrine_connection_pooling"] = true;
```

Comment:
It's awesome. Stolen from wikipedia:

```
In software engineering, a connection pool is a cache of database connections maintained so that the connections can
 be reused when future requests to the database are required.[citation needed] Connection pools are used to enhance
 the performance of executing commands on a database. Opening and maintaining a database connection for each user,
 especially requests made to a dynamic database-driven website application, is costly and wastes resources. In
 connection pooling, after a connection is created, it is placed in the pool and it is used again so that a new connection
 does not have to be established. If all the connections are being used, a new connection is made and is added to the
 pool. Connection pooling also cuts down on the amount of time a user must wait to establish a connection to the
 database.
```